content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases-node-cockroachdb.mdx

```diff
@@ -42,7 +42,7 @@ As a first step, navigate into your project directory that contains the `package
 Next, add the Prisma CLI as a development dependency to your project:
 
 ```terminal copy
-npm install prisma --save-dev
+npm install prisma --save-dev && npm install @prisma/client
```

Comment:
there shouldn't be any instructions in these files: `content/100-getting-started/02-setup-prisma/200-add-to-existing-project/...`

there's an explicit instruction to install the package on a later page: https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/install-prisma-client-typescript-postgresql

so we should remove the instructions from all of these pages.